src/ui/components/Header/ViewToggle.tsx

```diff
@@ -114,7 +114,7 @@ function ViewToggle({
 
   return (
     <AnimateSharedLayout type="crossfade">
-      <button className="view-toggle" type="button">
+      <div className="view-toggle" role="button">
```

Comment:
This change is because of line 93 in `common.css`:

```
/* Devtools rely on the old Gecko/HTML4 button hit testing */
html|button > * {
  pointer-events: none;
}
```

I don't understand that line, Chrome doesn't even seem to think it's a valid selector in prod, but for some reason it *does* pick it up in this branch locally? I'm not a huge fan of nesting content inside of buttons like this anyways, but I think maybe we should still delete that rule unless it's playing some important role that I don't understand.

Reply:
weird. such a strange line